public/javascripts/common/MissionStartTutorial.js

```diff
@@ -905,14 +905,26 @@ function MissionStartTutorial(missionType, labelType, data, svvOrsvl) {
             $('.label-on-image-type-icon').find('use').attr('xlink:href', iconID);
 
             $labelOnImage.show();
-        }
+        } 
 
         const $mstSlide = $('.mst-slide');
         const $labelTypeSubtitle = $('.label-type-subtitle');
         const $mstSlideImage = $('.msts-image');
         const $labelOnImage = $('.label-on-image');
         const $mstDoneButton = $('.mission-start-tutorial-done-btn');
 
+        const $labelOnImageDescription = $('.label-on-image-description');
+        const $languageButton = document.getElementById('language-button');
```

Comment:
Instead of checking for the language that's visible in the navbar, let's use the language parameter that we're already keeping track of and pass it along to `MissiontStartTutorial()`. In `explore.scala.html` we get the variable and pass it along to `Main.js`, and in `validation.scala.html` we pass it along to `Main.js` (they each have their own `Main.js`).